src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -422,141 +393,188 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia di 80#[#sym.mu]g su metro cubo ($80#sym.mu g\/m^3$), mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.5_Visualizzazione-tabella-superamento-soglia-inquinamento.png",width:70%),
+  caption: [UC4.5 Visualizzazione tabella superamento soglia di inquinamento dell'aria]
+)
+
+=== UC4.6: Visualizzazione tabella superamento soglia dei bacini idrici
+- *Attore Principale*: #glossary("amministratore pubblico")\;
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia del 70% di capienza, mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.6_Visualizzazione-tabella-superamento-soglia-bacini.png",width:70%),
+  caption: [UC4.6 Visualizzazione tabella superamento soglia dei bacini idrici]
+)
+
+=== UC4.7: Visualizzazione tabella superamento soglia delle zone ecologiche
+- *Attore Principale*: #glossary("amministratore pubblico")\;
+- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti la soglia del 80% di capienza, mostrando il valore superante la soglia, il #glossary("sensore") che ha rilevato tale valore e il timestamp relativo;
+- *Scenario Principale*:
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary[dashboard] superamento soglie.
+
+#figure(
+  image("assets/UML/UC4.7_Visualizzazione-tabella-superamento-soglia-cassonetti.png",width:70%),
+  caption: [UC4.7 Visualizzazione tabella superamento soglia delle zone ecologiche]
+)
+
+=== UC5: Ricezione notifiche di superamento soglie
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
 - *Postcondizioni*: l'#glossary("amministratore pubblico") riceve una notifica di superamento di una soglia impostata;
 - *Scenario Principale*:
   + Il #glossary("sistema") rileva condizioni che richiedono l'invio di una notifica per segnalare il superamento di una soglia impostata.
 #figure(
```

Comment:
```suggestion

```

Reply:
non ho capito

Reply:
vedi la conversazione prima, per poter spostare non me lo faceva fare in un unica suggestion.

Reply:
okok quello è risolto